tests/sftp/triggers/test_sftp.py

```diff
@@ -0,0 +1,16 @@
+from astronomer.providers.sftp.triggers.sftp import SFTPTrigger
+
+
+def test_sftp_trigger_serialization():
```

Comment:
can you please convert these tests to class-based tests. you can take a look at either core or cncf tests https://github.com/astronomer/astronomer-providers/blob/main/tests/core/sensors/test_external_task.py